sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -314,6 +314,8 @@ aditof::Status Adsd3500Sensor::open() {
             for (int i = 0; i < 10; i++) {
                 chipIDStatus = adsd3500_read_cmd(0x0112, &chipID);
                 if (chipIDStatus != Status::OK) {
+                    LOG(INFO) << "Could not read chip ID. Resetting ADSD3500 "
+                                 "to handle previous error.";
```

Comment:
Can we add this message to the dealias check in the next if statement as well?

Reply:
Is there a changes that it works to read chip ID but doesn't work to read dealias parameters?

Reply:
There might be.

Reply:
I have added the message. Thanks!